Tasks/Barys/Task3_1/query06.sql

```diff
@@ -0,0 +1,11 @@
+--Query 6
+SELECT TOP 10 
+    SOD.SalesOrderID,
+    SUM(SOD.UnitPrice * SOD.OrderQty) AS TotalPrice
```

Comment:
also it would be nice to add Discount to calculation (or use calculated column LineTotal)